lib/cli/src/generators/REACT_SCRIPTS/index.ts

```diff
@@ -20,10 +22,18 @@ const generator: Generator = async (packageManager, npmOptions, options) => {
       }
     : {};
 
-  await baseGenerator(packageManager, npmOptions, options, 'react', {
+  const craVersion = semver.coerce(
+    packageManager.retrievePackageJson().dependencies['react-scripts']
+  )?.version;
+  const isWebpack5 = semver.gte(craVersion, '5.0.0');
```

Comment:
```suggestion
  const isCra5 = semver.gte(craVersion, '5.0.0');
```

_Super_ minor, but the check is for the CRA version, not webpack.

Reply:
I'd suggest always coercing a version first, as this catches `-beta.0`, etc.
```ts
semver.gte(semver.coerce('42.6.7.9.3-alpha'))
```

Reply:
I don't mind the `isWebpack5` var. `isCra5` seems more logical - but then you might need to add a comment like "CRAv5 ships with WP5".